windows_docker_resources/Dockerfile.msvc2019

```diff
@@ -78,6 +78,35 @@ ENV OSPL_HOME C:\opensplice\HDE\x86_64.win64
 # You can't use ENV to append to the PATH https://stackoverflow.com/questions/42092932/appending-to-path-in-a-windows-docker-container
 RUN setx PATH "%PATH%;C:\Program Files\Git\cmd;C:\Program Files\CMake\bin;C:\OpenSSL-Win64\bin\;C:\xmllint\bin;"C:\opencv\x64\vc16\bin"
 
+# RTI Connext License
+COPY rticonnextdds-license\rti_license.dat C:\connext\
```

Comment:
What's Docker's behavior here?
* Will this always cache so that we need to bust it when the license file changes?
* Will this always copy and never cache?
* Will it copy only when the file has changed since the last time the build was invoked?

Reply:
My understanding is that the hash of the file contents are considered in the image, so I believe that your third bullet is the case.

Reply:
I agree with Scott, the rticonnextdds-license and rticonnextdds-src directories are part of the docker context. So long as these copied files hash to the same thing each time then these parts of the build can run from a cached image, until the submodule is updated and the files referenced change.